src/Tribe/Organizer.php

```diff
@@ -332,8 +332,12 @@ public function save_meta( $organizerId, $data ) {
 		 */
 		do_action( 'tribe_events_organizer_save', $organizerId, $data, $organizer );
 
-		if ( isset( $data['FeaturedImage'] ) && ! empty( $data['FeaturedImage'] ) ) {
-			update_post_meta( $organizerId, '_thumbnail_id', $data['FeaturedImage'] );
+		if ( isset( $data['FeaturedImage'] ) ) {
```

Comment:
Same here.

Reply:
I think the answer from above applies here as well.